components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.v1/src/main/java/org/wso2/carbon/identity/api/server/secret/management/v1/core/SecretManagementService.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org).
```

Comment:
Shall we fix this license?

Reply:
Addressed with commit [a6f4632](https://github.com/wso2/identity-api-server/pull/315/commits/a6f463218febb07c19f260f0aa26e33fdec47de3)